src/method/method_connection_sr.cpp

```diff
@@ -75,10 +79,14 @@ namespace cubmethod
     cubmem::block buffer (0, nullptr);
 
     int error = xs_receive_data_from_client (thread_p, &buffer.ptr, (int *) &buffer.dim);
-    if (error == NO_ERROR)
+    if (error == NO_ERROR && er_errid () == NO_ERROR)
       {
 	error = func (socket, buffer);
       }
+    else
+      {
+	error = (error == NO_ERROR) ? er_errid () : NO_ERROR; // ER_SP_TOO_MANY_NESTED_CALL
```

Comment:
If (error != NO_ERROR) are you trying to make it error = NO_ERROR?
Please check once more whether you are trying to process in the following meanings.
```
if(error == NO_ERROR) 
{
   error = er_errid ();
}
```

Reply:
Thank you. I've fixed it as your suggestion.